packages/syft/src/syft/core/node/common/client_manager/dataset_api.py

```diff
@@ -133,7 +133,7 @@ def all(self) -> List[Any]:
         result = [
             content
             for content in self.perform_api_request(
-                syft_msg=self._get_all_message
+                syft_msg=self._get_all_message, timeout=1
```

Comment:
Should we have a larger timeout? since it fetches all datasets so deserialization can take time if there are a large number of datasets uploaded.